opensrp-app/src/main/java/org/smartregister/AllConstants.java

```diff
@@ -485,6 +485,7 @@
         public static final String LOCATION_PICKER_TAG_SHOWN = "location.picker.tag.shown";
         public static final String ENCRYPT_SHARED_PREFERENCES = "encrypt.shared.preferences";
         public static final String ALLOW_OFFLINE_LOGIN_WITH_INVALID_TOKEN = "allow.offline.login.with.invalid.token";
+        public static final String ENABLE_FAST_SEARCH="enable.fast.search";
```

Comment:
Update the README with documentation for the new configuration

Reply:
Also maybe consider renaming this - what are we enabling exactly is it _a fast (More optimized search)_ or are we enabling _the Manual search button_ instead?

Reply:
Yeah I agree that name is a bit confusing. The idea was that the normal search is faster and would make sense in a device that can execute the current implementation with ease. `enable.manual.search.button` or something close would make sense